src/Parser/HavocDemonHunter/Modules/Features/AlwaysBeCasting.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Icon from 'common/Icon';
+
+import { formatPercentage } from 'common/format';
+
+import CoreAlwaysBeCasting from 'Parser/Core/Modules/AlwaysBeCasting';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class AlwaysBeCasting extends CoreAlwaysBeCasting {
+  static ABILITIES_ON_GCD = [
+
+    // Abilities
+    SPELLS.CHAOS_STRIKE.id,
+    SPELLS.BLADE_DANCE.id,
+    SPELLS.THROW_GLAIVE.id,
+    SPELLS.FURY_OF_THE_ILLIDARI.id,
+    SPELLS.EYE_BEAM.id,
+    SPELLS.DEATH_SWEEP.id,
+    SPELLS.ANNIHILATION.id,
+
+    // Talents
+    SPELLS.FELBLADE_TALENT.id,
+    SPELLS.FEL_ERUPTION_TALENT.id,
+    SPELLS.FEL_BARRAGE_TALENT.id,
+  ];
+
+  suggestions(when) {
+
+    const deadTimePercentage = this.totalTimeWasted / this.owner.fightDuration;
+
+    when(deadTimePercentage).isGreaterThan(0.25)
```

Comment:
This is a lot for a DPS.

Reply:
I agree. Gonna change it to 0.15 in the next PR.